components/view_image.jsx

```diff
@@ -198,7 +211,6 @@ export default class ViewImageModal extends React.Component {
                 content = (
                     <ImagePreview
                         fileInfo={fileInfo}
-                        fileUrl={fileUrl}
```

Comment:
Hey @saturninoabril @jwilander. 

I just moved this line into child component which I separated. Should I do this same for all other preview components, or restore?

Reply:
No need. If necessary, separate ticket will be created for it.